core-bundle/contao/themes/flexible/styles/main.css

```
@@ -1666,6 +1666,12 @@ table.tl_listing {
 	color: var(--gray);
 }
 
+.cte_type .drag-handle {
```

```suggestion
.cte_type .drag-handle {
	padding-block: 0;
```

That makes sure that the drag-handle won't vertically resize the row, it's small but noticeable (~ 2px)